### ISSUE TITLE
Some audio clips are either ~2 octaves too high or ~2 octaves too low

### ISSUE THREAD

**Issue by dippyshere** [dippyshere/Newest-Super-Human-Bros-PC#14]
Some sounds like starting a game are played at a higher pitch, and some sounds like the time reaching 100 and coin sound are played at a lower pitch than they should.

**Comment by stellaurora** [comment]
sdl_mixer issue most likely

**Comment by dippyshere** [comment]
ok

**Comment by dippyshere** [comment]
hey all, just wanted to provide an update on the issue. its resolved, however i havnt updated this game with a fix. the bug is in how sdl handles audio of different sample rates. [This reply to a reddit thread explains it better than i could](https://www.reddit.com/r/pygame/comments/dpg9ab/can_you_change_the_frequency_of_a_specific_channel/f5ymcbm?utm_source=share&utm_medium=web2x) but basically a double sample rate means double the length of a 1x of original. so sdl will slow down or speed up song, causing the pitch shift.

**Comment by dippyshere** [comment]
tl;dr
====
pretty much sample rate = data per second or whatever. more data per second, mean pygame has to speed up song to play every sample. less per second mean pygame slow down to play all to match the init() sample rate

**Comment by dippyshere** [comment]
fixed in https://github.com/dippyshere/Newest-Super-Human-Bros-PC/commit/3b31283190f43e40288509f9464cff242ce1c79b
